mungegithub/mungers/submit-queue.go

```diff
@@ -1310,7 +1387,7 @@ func (sq *SubmitQueue) doGithubE2EAndMerge(obj *github.MungeObject) bool {
 		return false
 	}
 
-	if !sq.e2eStable(true) {
+	if !sq.e2eStable(true) || obj.HasLabel(criticalFixLabel) {
```

Comment:
This should be `!sq.e2eStable(true) && !obj.HasLabel(criticalFixLabel)` or `!(sq.e2eStable(true) || obj.HasLabel(criticalFixLabel))`.